generators/base/generator-base.mjs

```diff
@@ -269,7 +263,7 @@ export default class JHipsterBaseGenerator extends PrivateBase {
    * @return {boolean} true if the entity is User.
    */
   isUserEntity(entityName) {
```

Comment:
Looks like this can be dropped, no uses.

Reply:
I guess it will be in some templates, kind of derived entity property

Reply:
I can be wrong, but I think I've already replaced every use of this api.

Reply:
Used indirectly in client/.../entity file update

Reply:
Done sir

Reply:
dropped :-)